pockerService.js

```diff
@@ -0,0 +1,31 @@
+const GetPocket = require("node-getpocket");
```

Comment:
Nice approach :-)

Reply:
Yeah, abstracting the actual network call away can help with mocking. Thus, the local development and testing can be much easier.